apps/predbat/predbat.py

```diff
@@ -301,8 +329,38 @@ def __init__(self, base, id=0, quiet=False):
         
     def update_status(self, minutes_now, quiet=False):
         """
-        Update inverter status
-        """
+        Update the following with inverter status.
+
+        Inverter Class Parameters
+        =========================
+
+            Parameter                          Type    Units    
+            ---------                          ----    -----    
+            self.rest_data                     bool
```

Comment:
rest_data is a dictionary containing the REST data read from the API, but you can do an IF on it as when it's empty it's false